### PR TITLE
Use custom test image for origin-aggregated-logging

### PR DESCRIPTION
**Pull request by vimalk78**
Use custom test image to have better control over packages required f…

### CODE REVIEW

**Comment by bparees** [comment]
looks right, let's see how the rehearsal job does

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml [hunk 1]
```diff
@@ -81,5 +84,6 @@ tests:
     from: logging-test-unit
 - as: smoke
   commands: make test
-  openshift_installer_src:
+  openshift_installer_custom_test_image:
     cluster_profile: aws
+    from: origin-aggregated-logging-tests
```
Comment (bparees) [inline ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]:
```suggestion
    from: stable:origin-aggregated-logging-tests
```

**Comment by bparees** [comment]
sorry i missed that you had the image ref wrong

/hold

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml (hunk 1 above)
Comment (vimalk78) [reply ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: thanks for correction. Updated

**Pull request merged**